src/phase0/phase0_fork_choice_spec.dfy

```diff
@@ -101,19 +101,18 @@ requires valid_time_slots_pure(store)
 requires valid_blocks_pure(store.blocks)
 {
   var target: Checkpoint := attestation.data.target;
-  var _ :- if !is_from_block then
+  if !is_from_block then
     validate_target_epoch_against_current_time_pure(store, attestation)
   else
```

Comment:
This actually changes the behaviour of [validate_on_attestation](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/fork-choice.md#validate_on_attestation).
The `validate_target_epoch_against_current_time_pure(store, attestation)` is an optional check, while the proposed change skips the rest of checks, when `!is_from_block`